client/my-sites/people/team-list/team.jsx

```diff
@@ -22,6 +26,54 @@ class Team extends React.Component {
 
 	renderLoadingPeople = () => <PeopleListItem key="people-list-item-placeholder" />;
 
+	getP2headerText = () => {
+		const { p2Guests, totalUsers, isP2HubSite, translate } = this.props;
+		const translateOptions = {
+			args: {
+				numberPeople: totalUsers,
+			},
+			count: totalUsers,
+			context: 'A navigation label.',
+		};
+
+		if ( isP2HubSite ) {
+			return translate(
+				'There is %(numberPeople)d member in this workspace',
+				'There are %(numberPeople)d members in this workspace',
+				translateOptions
+			);
+		}
+
+		if ( ! p2Guests?.found ) {
+			return translate(
+				'There is %(numberPeople)d member in this P2',
+				'There are %(numberPeople)d members in this P2',
+				translateOptions
+			);
+		}
+
+		const guests = translate( '%(numberGuests)d guest', '%(numberGuests)d guests', {
+			args: {
+				numberGuests: p2Guests.found,
+			},
+			count: p2Guests.found,
+		} );
+
+		const numMembers = totalUsers - p2Guests.found;
+		return translate(
+			'There is %(numMembers)d member and %(guests)s in this P2',
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?